content/docs/get-started/azure/modify-program.md

```diff
@@ -33,7 +33,9 @@ const account = new azure.storage.Account("storage", {
     resourceGroupName: resourceGroup.name,
     accountTier: "Standard",
     accountReplicationType: "LRS",
-    enableHttpsTrafficOnly: true,
+    tags: {
+        "Owner": "Example",
```

Comment:
Nit, but `Owner: Example` doesn't make too much sense... I usually do something like `Environment: Dev`, which also appears in some Microsoft docs. 
However, I googled and found an example of `Owner: x` in their docs too 😄